pkg/execution/executor/executor.go

```diff
@@ -471,14 +463,15 @@ func (e *executor) Schedule(ctx context.Context, req execution.ScheduleRequest)
 	//
 	// This enures that we only ever enqueue the start job for this function once.
 	queueKey := fmt.Sprintf("%s:%s", req.Function.ID, key)
+	retries := req.Function.GetRetries() + 1
```

Comment:
This feels a bit confusing, as we're setting `MaxAttempts` to `retries + 1`; should this be called `attempts` instead?
```suggestion
	attempts := req.Function.GetRetries() + 1
```